contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,286 @@
+  - Product engineers
+  - Growth
+  - Inside PostHog
+crosspost:
+  - Blog
+---
+
+People are signing up for your product. Great! But before you start planning your IPO, you need to make sure they actually stick around. This is the goal of **activation**.
+
+The trouble with activation is that it is hard to define. Many say it is when a user reaches the "aha" moment of your product, but this is vague and different for everyone.
+
+For example, we have 8+ products and each has a different set of activation metrics from launching a survey to inviting a teammate to creating two or more dashboards. What made finding activation metrics tricky was that each product had 10+ events that **could** be part of its activation metric.
+
+This means we spent a lot of time discovering our products' activation metrics. We're sharing our process here to help you do the same.
+
+But before we jump in... what is activation _really_?
+
+## What is an activation metric?
+
+Activation metrics are a set of actions a new user performs that are correlated with greater retention.
+
+Focusing on retention solves the trickiest part of defining an activation metric: it could be _anything_. Does a pageview count as activation? How about inviting a teammate? Creating their first object? Depending on your product, any or all of these could be an activation metric. It is **retention** that makes it clear.
+
+At PostHog, we define activation metrics as a set of actions where a company is much more likely to be using the product 3 months after starting. For [session replay](/session-replay), after analyzing multiple potential metrics, we found that watching 5 or more replays and setting a replay filter at least once were the actions that resulted in the highest retention rate.
+
+![Session replay activation metric](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_06_at_10_40_35_2x_a07c5ba4fd.png)
+
+Actions that don't make it into your activation metric can still be good, but they don't help you predict retention. For example, we want users to invite teammates, but we found that doing so doesn't mean they stick around, so optimizing this isn't the best use of our time.
+
+![Product analytics activation](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_13_at_09_53_01_2x_2c8bc81896.png)
+
+## How we found our activation metric
+
+How did we settle on watching 5 or more replays and setting a replay filter at least once? We tested a lot of different metrics by:
+
+1. Creating a big list of events that we thought might "hook" users into using our product.
+
+2. Creating groups of 3-5 events to test together. We aimed for 5-10 different groups, including the number of times the event occurs (e.g. watched 5 replays vs. watched 1 replay). Sometimes users have to do something more than once to reach their "ah-ha" moment.
+
+![Event groups](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_06_at_11_36_20_2x_89a95ebdf3.png)
```

Comment:
Right now this image is a lot of text. Perhaps some colours in the this image would make it easier to parse.

Reply:
How's this?

![CleanShot 2024-11-18 at 12 50 05@2x](https://github.com/user-attachments/assets/fe2efb8f-018a-4943-a1ef-72d7139fb836)

Reply:
Much better!